test/e2e_node/services/kubelet.go

```diff
@@ -194,6 +201,58 @@ func createPodManifestDirectory() (string, error) {
 	return path, nil
 }
 
+// createKubeconfig creates a kubeconfig file at the fully qualified `path`.
+func createKubeconfig(path string) error {
+	kubeconfig := `apiVersion: v1
+kind: Config
+users:
+- name: kubelet
+clusters:
+- cluster:
+    server: ` + getAPIServerClientURL() + `
+    insecure-skip-tls-verify: true
+  name: local
+contexts:
+- context:
+    cluster: local
+    user: kubelet
+  name: local-context
+current-context: local-context`
+
+	f, err := os.Create(path)
+	defer f.Close()
+	if err != nil {
+		return err
+	}
+	_, err = f.WriteString(kubeconfig)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func kubeconfigCWDPath() (string, error) {
+	cwd, err := os.Getwd()
```

Comment:
is the current working dir a good place for temp test files? I'd expect it to not be writeable sometimes, to result in a dirty git tree sometimes, and to conflict if multiple tests are run in parallel